docs/publish.rst

```diff
@@ -61,6 +61,15 @@ feeds, etc.) that you may have defined::
 
     pelican content -s publishconf.py
 
+To base your publish configuration on top of your ``pelicanconf.py``, you
+can import your ``pelicanconf`` settings by including in ``publishconf.py``
+the line::
```

Comment:
you can import your pelicanconf settings by including the following line in your publishconf